packages/react-hooks/src/useRefEffect.ts

```diff
@@ -35,31 +35,34 @@ export function useRefEffect<T>(callback: (value: T) => (() => void) | void, ini
     callback: (value: T) => (() => void) | void;
     cleanup?: (() => void) | void;
```

Comment:
Remove `RefData` since you're not really using it anymore. You can explicitly type `cleanup` below.